backend/src/api/user-login.js

```diff
@@ -12,30 +12,32 @@ const login = async (req, res) => {
       return res.status(404).json({ message: "User Does Not Exist" });
     }
 
-    const isPasswordCorrect = await bcrypt.compare(
-      password,
-      existingUser.password
-    );
+    const isPasswordCorrect = await bcrypt.compare(password, existingUser.password);
 
     if (!isPasswordCorrect) {
       return res.status(400).json({ message: "Invalid Password" });
     }
 
-    const token = jwt.sign(
-      {
+    const token = jwt.sign({
         _id: existingUser._id,
         name: existingUser.name,
         email: existingUser.email,
         password: existingUser.password,
+        tokens: existingUser.tokens
       },
       "test",
-      { expiresIn: "1h" }
-    );
+      {expiresIn:"1h"});
 
-    res.status(200).json({ token });
+    res.cookie('token', 'Bearer ' + token, {
```

Comment:
Use http cookies for auth since it cannot be read or altered in the browser / console